playwright-e2e/pages/dsm/kitsInfo-pages/kitsReceived-page/kitsReceivedPage.ts

```diff
@@ -0,0 +1,95 @@
+import {KitType} from 'lib/component/dsm/kitType/kitType';
+import {KitsTable} from 'lib/component/dsm/tables/kitsTable';
+import {APIRequestContext, expect, Page} from '@playwright/test';
+import {KitTypeEnum} from 'lib/component/dsm/kitType/enums/kitType-enum';
+import {waitForNoSpinner, waitForResponse} from 'utils/test-utils';
+import {KitsColumnsEnum} from '../enums/kitsColumns-enum';
+
+const { BSP_TOKEN, DSM_BASE_URL } = process.env;
+
+export default class KitsReceivedPage {
+  private readonly PAGE_TITLE = 'Kits Received';
+  private readonly kitType = new KitType(this.page);
+  private readonly kitsTable = new KitsTable(this.page);
+
+  constructor(private readonly page: Page,
+              private readonly request: APIRequestContext) {
+  }
+
+  public async waitForLoad(): Promise<void> {
+    await this.page.waitForLoadState('networkidle');
```

Comment:
This is not an reliable way for synchronization. I'm good with keeping it for now because I don't have a better suggestion at this time.